openstack/networking/v2/subnets/testing/requests_test.go

```diff
@@ -276,6 +277,7 @@ func TestCreate(t *testing.T) {
 		NetworkID: "d32019d3-bc6e-4319-9c1d-6722fc136a22",
 		IPVersion: 4,
 		CIDR:      "192.168.199.0/24",
+		GatewayIP: gophercloud.MaybeString("192.168.199.1"),
```

Comment:
`MaybeString` is intended for internal use (but created before an `internal` package was a thing). Because users refer to tests for direction, let's define an IP address as a constant (or variable, if it'll get changed) and take its address here.